src/test/java/io/jenkins/plugins/gitlabbranchsource/GitLabHookCreatorParameterizedTest.java

```diff
@@ -1,4 +1,4 @@
-package io.jenkins.plugins.gitlabbranchsource;
+ackage io.jenkins.plugins.gitlabbranchsource;
```

Comment:
Need to revert this change

```suggestion
package io.jenkins.plugins.gitlabbranchsource;
```

Reply:
Done, thanks for the feedback